README.md

```diff
@@ -35,3 +35,10 @@ Refer to [docs/changelog.md](docs/changelog.md)
 ## Contributing to NEURON development
 
 Refer to [NEURON contribution guidelines](CONTRIBUTING.md)
+
+## Funding
+
+NEURON is developed by Yale University, supported by funding NIH grant number R01NS11613 (Yale University). 
```

Comment:
The wording here is not quite right but I'm not sure how to fix. I'll discuss with colleagues.

Reply:
"NEURON is developed by Yale University" is actually incorrect. Instead would suggest 
NEURON development is supported by NIH grant R01NS11613
Might consider 
NEURON development is supported by NIH grant R01NS11613 (PI M.L. Hines at Yale University)
but would that trigger an avalanche of other changes?

Reply:
> NEURON development is supported by NIH grant R01NS11613

Seems clear and concise. 
Which other changes do you think would need to be made in this case?